indico_zodbimport/modules/event_abstracts.py

```diff
@@ -69,7 +69,7 @@ def strict_sanitize_email(email, fallback=None):
 
 
 # Add "temporary" column to model
-Contribution.legacy_track_id = db.Column(db.Integer, nullable=True)
+# Contribution.legacy_track_id = db.Column(db.Integer, nullable=True)
```

Comment:
Yes. That basically breaks the whole thing since the column doesn't exist anymore. I decided to leave it commented so that we read it when rewriting the migration scripts.